shopinvader/models/shopinvader_backend.py

```diff
@@ -146,12 +146,14 @@ def bind_all_category(self):
     def _send_notification(self, notification, record):
         self.ensure_one()
         record.ensure_one()
-        notif = self.env['shopinvader.notification'].search([
+        notifs = self.env['shopinvader.notification'].search([
             ('backend_id', '=', self.id),
             ('notification_type', '=', notification),
-            ])
-        if notif:
-            notif.with_delay().send(record.id)
+        ])
+        description = "Notification %s for %s,%s" % (
```

Comment:
@acsonefho 
cosmetic :smirk: 
s/Notification/Notify
In the same time the message could be translated `_(" ")% (...)`